datahub-frontend/build.gradle

```diff
@@ -26,4 +30,20 @@ task myTar(type: Tar) {
 
 artifacts {
   archives myTar
+}
+
+graphqlCodegen {
```

Comment:
If we can add these files via gradle task, why do we need to check the generated classes into git? Can we just gitignore them?